src/figuresClasses.ts

```diff
@@ -1,19 +1,86 @@
-export interface Figure {
+type FigureType = 'circle' | 'rectangle' | 'triangle';
+type FigureColor = 'red' | 'green' | 'blue';
 
+export interface Figure {
+  shape: FigureType;
+  color: FigureColor;
+  getArea(): number;
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  shape: FigureType = 'triangle';
```

Comment:
not necessary because you implement Figure and there you declared it 
```suggestion
  shape = 'triangle';
```